patches/minecraft/net/minecraft/world/entity/npc/VillagerTrades.java.patch

```diff
@@ -0,0 +1,22 @@
+--- a/net/minecraft/world/entity/npc/VillagerTrades.java
++++ b/net/minecraft/world/entity/npc/VillagerTrades.java
+@@ -1367,9 +_,6 @@
+         private final int villagerXp;
+ 
+         public EmeraldsForVillagerTypeItem(int p_35669_, int p_35670_, int p_35671_, Map<VillagerType, Item> p_35672_) {
+-            BuiltInRegistries.VILLAGER_TYPE.stream().filter(p_35680_ -> !p_35672_.containsKey(p_35680_)).findAny().ifPresent(p_327046_ -> {
+-                throw new IllegalStateException("Missing trade for villager type: " + BuiltInRegistries.VILLAGER_TYPE.getKey(p_327046_));
+-            });
```

Comment:
Instead of patching out this entire validation, you should add a namespace for "minecraft" check in the filter.

Reply:
Yup (We just talked about this lol)